include/hashinator/hashinator.h

```diff
@@ -483,7 +515,7 @@ class Hashmap {
 
 #ifdef HASHINATOR_CPU_ONLY_MODE
    void clear() {
-      buckets = split::SplitVector<hash_pair<KEY_TYPE, VAL_TYPE>>(1 << _mapInfo->sizePower, {EMPTYBUCKET, VAL_TYPE()});
+      buckets = split::SplitVector<hash_pair<KEY_TYPE, VAL_TYPE>,Allocator>(1 << _mapInfo->sizePower, {EMPTYBUCKET, VAL_TYPE()});
```

Comment:
shouldn't this also take `_allocator` as an argument? (It could still be a custom CPU-only allocator)